packages/eslint-plugin-pf-codemods/lib/rules/v5/wizard-update-deprecatedPath.js

```diff
@@ -2,13 +2,22 @@ const { moveSpecifiers } = require("../../helpers");
 
 const specifiersToMove = [
   "Wizard",
+  "WizardProps",
+  "WizardStep",
   "WizardContext",
+  "WizardContextType",
```

Comment:
Do we want to include these even though they are not included in `index.ts`?

Reply:
Since they're being exported from the Context file that's included in index it wouldn't hurt. We're doing something similar in the Select deprecation codemod